packages/es-components/src/components/containers/horizontalScrollWrapper/HorizontalScrollWrapper.js

```diff
@@ -0,0 +1,225 @@
+import React, { useRef, useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import Icon from '../../base/icons/Icon';
+import withWindowSize from '../../util/withWindowSize';
+
+const OuterWrapper = styled.div`
+  overflow: hidden;
+  position: relative;
+`;
+
+const InnerWrapper = styled.div`
+  alignitems: center;
+  display: flex;
+  height: 100%;
+  position: relative;
+  white-space: nowrap;
+  z-index: 0;
+`;
+
+const ScrollIcon = styled.div`
+  align-items: center;
+  cursor: pointer;
+  display: flex;
+  height: 100%;
+  outline: 0;
+  position: absolute;
+  top: 0;
+  z-index: 1;
+`;
+
+const ArrowContainer = styled.div`
+  align-items: center;
+  background-color: ${props => props.theme.colors.white};
+  border-radius: 50%;
+  box-shadow: 0 1px 1px ${props => props.theme.colors.boxShadowDark};
+  color: ${props => props.theme.colors.gray9};
+  display: flex;
+  padding: 0.5em;
+`;
+
+const generateOnArrowKeyDownHandler = onKeyDown => event => {
+  const code = event.code.toLowerCase();
+  if (code === 'Enter' || code === 'Space') {
```

Comment:
This will never be true because we lower case it on the line above.

Reply:
Very true! Fixed!